database/factories/UserFactory.php

```diff
@@ -14,26 +16,14 @@ class UserFactory extends Factory
      */
     public function definition()
     {
+        $faker = Faker::create();
+
         return [
-            'name' => $this->faker->name(),
-            'email' => $this->faker->unique()->safeEmail(),
-            'email_verified_at' => now(),
-            'password' => '$2y$10$92IXUNpkjO0rOQ5byMi.Ye4oKoEa3Ro9llC/.og/at2.uheWG/igi', // password
-            'remember_token' => Str::random(10),
+            'name' => $faker->name,
+            'mail_address' => $faker->unique()->safeEmail,
```

Comment:
Phần này sao em không dùng luôn $this->faker và đổi qua dùng $faker biến riêng bên trên làm gì??